### ISSUE TITLE
Relocate Microsoft.ML.TensorFlow.TestModels nuget or grant access to dotnet-core feed.

### ISSUE THREAD

**Issue by zeahmed** [dotnet/machinelearning#2145]
Currently, Microsoft.ML.TensorFlow.TestModels nuget is created from https://github.com/dotnet/machinelearning-testdata and push to dotnet-core feed. 

ML.Net people don't have push access to dotnet-core feed. Whenever there is a change in the repo and nuget needs to be regenerated, we need to ask the relevant people (who have access) to do this. The solution is either create a new feed where ML.Net people have access to or grant push access on dotnet-core to ML.Net people (whichever is convenient).

Cc: @yaeldekel, @ericstj, @eerhardt.

**Comment by eerhardt** [comment]
Another possible solution would be to set up an “official build” for the repo that built the packages on new merges and pushed them to the NuGet feed, just like the ML.NET official build does.

**Comment by ericstj** [comment]
Yeah, and build on changes like we do with other support repositories.

**Comment by zeahmed** [comment]
Also, need access to update https://www.nuget.org/packages/Microsoft.ML.TensorFlow.Redist/
Version 1.12 of TensorFlow is out and we need to update this nuget with latest version.

**Comment by rosebyte** [comment]
`Microsoft.ML.TensorFlow.TestModels` is now hosted on the dedicated `machinelearning-assets` feed and is no longer consumed from `dotnet-core`.
